lib/pbench/test/unit/server/test_requests.py

```diff
@@ -353,7 +353,7 @@ def test_upload(
         assert dataset.name in self.cachemanager_created
 
         for record in caplog.records:
-            assert record.levelname in ["DEBUG", "INFO"]
+            assert record.levelname in ["DEBUG", "INFO", "WARNING"]
```

Comment:
Pretty sure we shouldn't expect "WARNING" unless we're deliberately testing edge conditions. I believe both of your new `warning` logs should go away or at least be downgraded to `info` for short-term debugging purposes.